examples/cluster/plot_bisect_kmeans.py

```diff
@@ -6,9 +6,13 @@
 This example shows differences between Regular K-Means algorithm and Bisecting K-Means.
 
 While K-Means clusterings are different when with increasing n_clusters,
```

Comment:
I am not convinced by the current wording. In both K-means and Bisecting K-means cases the clusters are "different".
```suggestion
While K-Means models are independent among them when increasing n_clusters,
```

Reply:
I don't have the "model" in mind in this sentence, but the actual clusterings created.

Reply:
Feel free to rephrase. The important thing to remark is the "independent clusters" more than the "different clusters"